### DISCUSSION TITLE
Installing on 2.6.5/2.6.6

### DISCUSSION THREAD

**Discussion by Tomi6800** [xblax/flashforge_ad5m_klipper_mod#169] [Show and tell]
Beim Flashforge 5M Pro Hut-Die-Installations-Lauf-Lauf-, siehe Bild. Firmware 2.6.6-2.3.3 
![Screenshot](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/168713673/b1409c0a-46e9-4bb0-9563-c714d70cc3c7)

**Comment by consp** [comment]
This is a known issue and will be fixed in the next release. It won't install on anything except 2.4.5. See https://github.com/xblax/flashforge_ad5m_klipper_mod/discussions/157#discussioncomment-9209886 for a hint on how to install it if you do not want to wait (it requires extracting the tgz file to the usb drive and removing some lines from the install script.)

See also #144 #147 #153 #157

(I'm not responding in German as this is meant for an international audience).
